code/game/objects/items/sticker/sticker.dm

```diff
@@ -0,0 +1,192 @@
+/obj/item/sticker
+	name = "sticker" //Don't bother changing the name or desc for subtypes
+	desc = "An adhesive graphic."
+	icon = 'icons/obj/sticker.dmi'
+	icon_state = "happy"
+	flags_1 = IS_ONTOP_1
+	w_class = WEIGHT_CLASS_TINY
+	appearance_flags = TILE_BOUND | PIXEL_SCALE | KEEP_APART
+	///Our current state for being stuck or unstuck
+	var/sticker_state = STICKER_STATE_ITEM
+	///Built appearance for item state
+	var/mutable_appearance/item_appearance
+	///Build appearance for stuck state
+	var/mutable_appearance/stuck_appearance
+	///Sticker icon
+	var/sticker_icon
+	var/sticker_icon_state = "happy_sticker"
+	///Do we add an outline?
+	var/do_outline = TRUE
+	///What kind of tint we rocking?
+	var/tint = "#fff"
+	///Sticker flags
+	var/sticker_flags
```

Comment:
do not merge rarity flags and series flag.
but also, rarity doesn't need to be bitflags.